src/components/BlogPost/BlogPost.jsx

```diff
@@ -29,9 +29,16 @@ export default function BlogPost({
     coverImagePhotographerName,
   } = frontmatter;
 
+  const coverImageUrl = `https://source.unsplash.com/${coverImageUnsplashId}/624x384`;
+
   return (
     <Layout>
-      <SEO title={`Blog: ${title}`} />
+      <SEO
+        title={`Blog: ${title}`}
```

Comment:
I think I agree with you, it does look weird with the `Blog: ` prefix.
Let's remove it! 👍 

```suggestion
        title={title}
```